inspirehep/modules/workflows/serializers/json.py

```diff
@@ -0,0 +1,32 @@
+from os.path import basename
+
+from flask import json
+
+from invenio_workflows_ui.serializers.json import JSONSerializer
+from inspire_utils.record import get_value
+from inspirehep.utils.record_getter import get_db_record
+
+
+class WorkflowJSONSerializer(JSONSerializer):
+    def serialize(self, workflow_ui_object):
+        fuzzy_matches = get_value(workflow_ui_object, '_extra_data.matches.fuzzy')
+
+        if fuzzy_matches:
+            workflow_ui_object['_extra_data']['matches']['fuzzy'] = \
+                map(self._ref_to_literature_brief, fuzzy_matches)
+
+        return json.dumps(workflow_ui_object.dumps(), **self._format_args())
+
+    def _ref_to_literature_brief(self, record_ref):
+        recid = basename(record_ref['$ref'])
```

Comment:
we should have a test case with malformed ``$ref``, for example ``/api/record/``

Reply:
so this means we have to handle it too. How do you propose to handle this case?